src/Xamarin.Android.Tools.Aidl/Xamarin.Android.Tools.Aidl.csproj

```diff
@@ -7,6 +7,8 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Xamarin.AndroidTools.Aidl</RootNamespace>
     <AssemblyName>Xamarin.Android.Tools.Aidl</AssemblyName>
+    <ProductVersion>8.0.30703</ProductVersion>
```

Comment:
If possible, avoid "meaningless" changes to files you're not otherwise changing. (Yes, the IDE did it, and yes, you can instead `git checkout -f HEAD ...` these files...)